docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -7,15 +7,15 @@
 [source,xml,indent=0]
 ----
 <dependencyManagement>
-  <dependencies>
-    <dependency>
-      <groupId>com.azure.spring</groupId>
-      <artifactId>spring-cloud-azure-dependencies</artifactId>
-      <version>4.0.0-beta.2</version>
-      <type>pom</type>
-      <scope>import</scope>
-    </dependency>
-  </dependencies>
+    <dependencies>
```

Comment:
I tested it, use 2 spaces as indent is better in the web page, could you please revert this?